src/dataExplorer/components/resources/types/script/index.ts

```diff
@@ -1,6 +1,7 @@
 import {ResourceType} from 'src/types/resources'
 import editor from './editor'
 import {CLOUD} from 'src/shared/constants'
+import {DEFAULT_EDITOR_TEXT} from 'src/dataExplorer/context/persistance'
```

Comment:
The changes in this file should be the only change required to enact this change. The other places where you set the query then navigate to call this code

Reply:
If I only make the changes in the script resource then we have a missing message on first load of the page.

see that I demonstrate this with the commit (`test: demonstrate that we need the default query set on both the session, and in the script resource. In order to have on-load provide the default message.`), and then made a video of the behavior (refer to the PR convo).

Reply:
makes sense